### PR TITLE
Number calculations using program arguments

### LINKED ISSUE #1: Task description
image

### PR DESCRIPTION
**Pull request by wojciechkostecki**
Closes #1

### CODE REVIEW

**Comment by rbobko93** [comment]
Use better name for the PR. It should be descriptive just like commit messages. @wojciechkostecki

File: src/Main.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    private static void squaredNumbers(List<Integer> list) {
+        List<Integer> squared = list.stream()
+                .map(x -> x * x)
+                .collect(Collectors.toList());
+        System.out.println(squared);
+    }
+
+    private static void printAllNumbersAndSumNumbers(List<Integer> list) {
```
Comment (rbobko93) [inline src/Main.java]: I would change method name to something shorter, like `printNumberSum`

**Review by rbobko93 — CHANGES_REQUESTED** [review]

**Comment by rbobko93** [comment]
@wojciechkostecki this PR name is not descriptive at all. It should say what are the changes in summary.

File: src/Main.java (hunk 1 above)
Comment (rbobko93) [reply src/Main.java]: @wojciechkostecki when you resolved issue from a comment mark conversation as resolved with button below: "Resolve conversation"

**Review by rbobko93 — CHANGES_REQUESTED** [review]
Change PR name and resolve conversation about method name

File: src/Main.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+        System.out.println(numbersList.size());
+        System.out.println(numbersList);
+
+        squaredNumbers(numbersList);
+
+        printNumberSum(numbersList);
+    }
+
+    private static void squaredNumbers(List<Integer> list) {
```
Comment (rbobko93) [inline src/Main.java]:
```suggestion
    private static void printSquaredNumbers(List<Integer> list) {
```

**Review by rbobko93 — CHANGES_REQUESTED** [review]
Change PR name, how many times do I have to say this?

**Review by rbobko93 — CHANGES_REQUESTED** [review]
Saying this for the 5th time now, change PR name!

**Comment by wojciechkostecki** [comment]
> Saying this for the 5th time now, change PR name!


You suggested that name.

**Comment by wojciechkostecki** [comment]
> > Saying this for the 5th time now, change PR name!
> 
> You suggested that name.

image

**Comment by rbobko93** [comment]
> > > Saying this for the 5th time now, change PR name!
> > 
> > 
> > You suggested that name.
> 
> image

The PR name is still Develop. You added my name as a comment to the PR. Also assign this PR to yourself. Top right corner below Reviewers.

**Comment by wojciechkostecki** [comment]
> > > > Saying this for the 5th time now, change PR name!
> > > 
> > > 
> > > You suggested that name.
> > 
> > 
> > image
> 
> The PR name is still Develop. You added my name as a comment to the PR. Also assign this PR to yourself. Top right corner below Reviewers.

Done

**Comment by rbobko93** [comment]
> > > > > Saying this for the 5th time now, change PR name!
> > > > 
> > > > 
> > > > You suggested that name.
> > > 
> > > 
> > > image
> > 
> > 
> > The PR name is still Develop. You added my name as a comment to the PR. Also assign this PR to yourself. Top right corner below Reviewers.
> 
> Done

You didn't assign yourself to this review :| Read carefully what I am saying in my comments and don't try to rush things. Approving because this already took too long.

**Review by rbobko93 — APPROVED** [review]

**Pull request merged**